src/plugins/dam/components/image-picker-field/ImagePickerModal.jsx

```diff
@@ -22,6 +22,10 @@ export default function ImagePickerModal(props) {
   const tab = allowLinked ? 'linked-images' : 'search-images'
   const [activeTab, setActiveTab] = useState(tab);
 
+  const selectActiveTab = (tab) => {
```

Comment:
@meghedi whats the advantage of using this? seems like you could just inline it

Reply:
When there is not linked images, there is a "search" button that goes to search-images tab, selectActiveTab prop needs to be sent to linkedImages in that case.  
![Screen Shot 2022-10-20 at 11 19 31 AM](https://user-images.githubusercontent.com/5440662/197027474-ec71a6c6-b55c-491d-9445-74bf4bc7e8e8.png)